server/proxy/bindings.go

```diff
@@ -136,6 +137,18 @@ func (p *Proxy) scanAppBindings(app *apps.App, bindings []*apps.Binding, locPref
 			continue
 		}
 
+		if fql.In(apps.LocationCommand) {
+			label := b.Label
+			if label == "" {
+				label = string(b.Location)
+			}
+
+			if strings.ContainsAny(label, " \t") {
+				// p.mm.Log.Debug("Binding validation error: Command label has multiple words", "app", app.Manifest.AppID, "location", b.Location)
```

Comment:
1/5 that we should log an error

Reply:
I left it as a comment because we are not logging also a message a few lines above when the app has not allowed that binding. I can uncomment both, but I fear a logs flood in case of a "not so well developed app". What do you think?

Reply:
Given that apps are in "developers preview" right now, developers should be our focus group. Hence, if the log messages help devs, I think we should add them.